release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
+This release adds a check to ensure that disks are UEFI-compatible before enabling features that require UEFI, such as secure boot. 
+As a result, you can use non-UEFI disks if needed. 
+This change adds new permissions requirements for `compute.images.get` and `compute.images.getFromFamily`. (link:https://issues.redhat.com/browse/OCPBUGS-44671[OCPBUGS-44671])
+
+* Previously, when the {aws-short} DHCP option set was configured to use a custom domain name that contains a trailing period (`.`), {product-title} installation failed. 
+With this release, the logic that extracts the hostname of EC2 instances and turns them into kubelet node names trims trailing periods so that the resulting Kubernetes object name is valid. 
+Trailing periods in the DHCP option set no longer cause installation to fail. (link:https://issues.redhat.com/browse/OCPBUGS-45306[OCPBUGS-45306])
+
+* Previously, the number of {azure-short} availability set fault domains used a fixed value of `2`. 
+This setting works in most {azure-short} regions because fault domain counts are typically at least 2. 
+However, this setting failed in the `centraluseuap` and `eastusstg` regions. 
+With this release, the number of availability set fault domains in a region is set dynamically. 
+(link:https://issues.redhat.com/browse/OCPBUGS-45663[OCPBUGS-45663])
+
+* Previously, the {azure-short} cloud controller manager panicked when there was a temporary API server disconnection. 
+With this release, the {azure-short} cloud controller manager correctly recovers from temporary disconnection. 
+(link:https://issues.redhat.com/browse/OCPBUGS-45859[OCPBUGS-45859])
+
+* Previously, some services became stuck in a pending state due to incorrect or missing annotations. 
+With this release, validation added to the {azure-short} `service.beta.kubernetes.io/azure-load-balancer-tcp-idle-timeout` and {gcp-short} `cloud.google.com/network-tier` annotations resolves the issue. 
+(link:https://issues.redhat.com/browse/OCPBUGS-48481[OCPBUGS-48481])
+
+* Previously, the method used to fetch the provider ID from {aws-short} could fail to provide this value to the kubelet when needed. 
+As a result, sometimes machines could get stuck in different states and fail to complete initialization. 
+With this release, the provider ID is consistently set on kubelet startup. 
+(link:https://issues.redhat.com/browse/OCPBUGS-50905[OCPBUGS-50905])
+
+* Previously, an incorrect endpoint in the {azure-short} cloud controller manager caused installations on {azure-full} Government Cloud to fail. 
+The issue is resolved in this release.
+(link:https://issues.redhat.com/browse/OCPBUGS-50969[OCPBUGS-50969])
+
+* Previously, the Machine API sometimes detected an unhealthy control plane node during cluster creation on {ibm-cloud-title} and attempted to replace the node. 
+This effectively destroyed the cluster. 
+With this release, the Machine API only attempts to replace unhealthy compute nodes during cluster creation. 
```

Comment:
```suggestion
With this release, the Machine API only attempts to replace unhealthy compute nodes during cluster creation and does not attempt to replace unhealthy control plane nodes. 
```
just because I had to read that twice